### PR TITLE
fix(env): fix tfm-test script

### PR DESCRIPTION
**Pull request by amirfefer**
## Updated Packages

* [ ] root
* [ ] @theforeman/builder
* [x] @theforeman/env
* [ ] @theforeman/eslint-plugin-foreman
* [ ] @theforeman/vendor
* [ ] @theforeman/vendor-dev
* [ ] @theforeman/vendor-core

## PR Type

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] Other… Please describe:

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

## Do you use this PR in another repository?

* [ ] Yes
* [ ] No

### CODE REVIEW

File: packages/env/test/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+  presets: [
+    require.resolve('@theforeman/builder/babel', {
```
Comment (sharvit) [inline packages/env/test/babel.config.js]: Are you sure `require.resolve('@theforeman/builder/babel')` won't be enough? (now you have it as a dep instead peerDep so it should fix it?)

File: packages/env/package.json [hunk 2]
```diff
@@ -23,20 +23,17 @@
     "tfm-test": "./bin/tfm-test.js"
   },
   "dependencies": {
+    "@theforeman/builder": "^3.8.3",
```
Comment (sharvit) [inline packages/env/package.json]: Should be `^3.9.0`

Comment (amirfefer) [reply packages/env/package.json]: bumped to 3.9.0

**Pull request merged**